pkg/shell/indexes.js

```diff
@@ -213,12 +214,9 @@ function MachinesIndex(index_options, machines, loader) {
             compiled = compile(machine);
 
         ReactDOM.render(
-            React.createElement(TopNav, {
-                index: index,
-                state: state,
-                machine: machine,
-                compiled: compiled,
-            }),
+            <WithDialogs>
+                <TopNav index={index} state={state} machine={machine} compiled={compiled} />
```

Comment:
The file should be renamed to indexes.jsx then. Not that our build chain would care, but for consistency, at least until we decide to just rename all *.jsx files to *.js.

Reply:
Done.

Reply:
Done.